### DISCUSSION TITLE
Thanks to Jisoo!

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [JisooSeo/101-WG-Discussions#8] [General]
I just wanted to thank Jisoo for creating this great resource for us Ontology 101 WG members! I'll probably post a Q&A soon concerning a couple of problems I had with Protege while working through the Pizza Ontology tutorial, but until then, I've been enjoying reading through the discussions that have already been posted.

**Comment by JisooSeo** [comment]
Thank you so much for your kind message, Greg! Please post away 👍 Meanwhile, I will update the discussion section soon so we have more resources and Q&A 😄

**Comment by johnbeve** [comment]
Many thanks indeed!
